tests/src/App/ComplexGeoData.cpp

```diff
@@ -74,9 +74,9 @@ class ComplexGeoDataTest: public ::testing::Test
     {
         if (App::Application::GetARGC() == 0) {
             constexpr int argc = 1;
-            std::array<char*, argc> argv {"FreeCAD"};
```

Comment:
This block is copied in several places in the tests directory. We should move this to a separate header file. I will prepare a separate PR.